pmlc/dialect/stdx/ir/ops.td

```diff
@@ -42,4 +42,26 @@ def AtomicRMWYieldOp :
   let arguments = (ins AnyType:$result);
 }
 
+def FPToSIOp :
+    StdX_Op<"fptosi"> {
+  let summary = "cast from floating-point to signed integer type";
+  let arguments = (ins AnyType:$result);
+  let results = (outs AnyType);
```

Comment:
Use something like `AnyFloatType` for the arguments and `AnyIntegerType` for the result.